script.js

```diff
@@ -0,0 +1,129 @@
+var a, b, param,c ;
+var a4;
+
+
+
+// Отлавливается знак и проиходят вычисления
+function mathOperation(){
+    switch(param){
+        case '+':            
+            c = a + b;
+            break;
+        case '-':            
+            c = a - b;
+            break;  
+        case '/':            
+            c = a / b;
+            break;            
+        case '*':
+            c = a * b;
+            break;     
+        default:            
+            alert('Некорректное значение') ;  
+            enterNumbers();
+            break;
+            
+    }
+}
+
+
+
+
+ // Вводятся числа, идет проверка на числа, запускается функция mathOperation()
+function enterNumbers(){
+    a = +prompt('Введите числи А');
+    b = +prompt('Введите числи Б'); 
+    param = prompt('Введите параметр');
+    if( isNaN(a) || isNaN(b)){
```

Comment:
Не забывайте о пробелах между операторами и операндами
```suggestion
    if( isNaN(a) || isNaN(b)) {
```